README.md

```diff
@@ -5,7 +5,7 @@ Status](https://travis-ci.org/ministryofjustice/c100-application.svg?branch=mast
 
 Work in progress: This is a Rails application to enable citizens
 to complete the C100 form. It is based on software patterns developed for the
-[Appeal to the Tax Tribunal][taxtribs] application.
+[Appeal to the Tax Tribunal][taxtribs] application. This iteration of the app has been created as a learning project to be considered independently of the original app.
```

Comment:
Please use the text provided in the original github issue, and add it as a new first paragraph, rather than adding it *to* the first paragraph.